index.js

```diff
@@ -6,9 +6,10 @@ const clonedeep = require('lodash.clonedeep')
  *
  * @param {object} cypressConfig - The cypress config object
  * @param {object} dotEnvConfig - (optional) The dotenv config object, ref: https://www.npmjs.com/package/dotenv#config
+ * @param {string} all - (optional) Whether to return all env variables. If set to false (default), only env variables prefixed with CYPRESS_ are returned.
  * @returns {object} The cypress config with an augmented `env` property
  */
-module.exports = (cypressConfig, dotEnvConfig) => {
+module.exports = (cypressConfig, dotEnvConfig, all = false) => {
```

Comment:
love this new parameter! that way it keeps the existing behavior and won't break stuff for others.